File: code/python/removeDuplicates.py
class Solution:
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if(len(nums)<=1):
            print(len(nums))
            return len(nums)
        i = 0
        while i < len(nums)-1:
            if nums[i] == nums[i+1]:
                nums.pop(i+1)
            else:
                i = i+1
        return len(nums)

File: code/java/removeDuplicates.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums.length < 2)
            return(nums.length);
        int i = 0;
        int index = 0;
        while(i<nums.length){
            if(nums[i] == nums[index])
                i++;
            else
                nums[++index] = nums[i++];
        }
        nums = Arrays.copyOfRange(nums, 0, index+1);
        return nums.length;
    }
}
